docs/BUILD.md

```diff
@@ -115,6 +115,14 @@ To build the code, you just need to run `build.ps1` in the `scripts` folder:
 ./scripts/build.ps1
 ```
 
+Note that in windows you will need to use Powershell 7 to create the build and Powershell 7 (in administrator mode) to run the tests.
```

Comment:
It already states above that you need powershell. I don't think you need anything specific to tests in this document.